metadataStore/commands.py

```diff
@@ -276,30 +241,34 @@ def find_header(limit=50, **kwargs):
         pass
 
     try:
-        st_time_dict = kwargs.pop('create_time')
-        if not isinstance(st_time_dict, dict):
-            raise ValueError('Wrong format. create_time must include '
+        time_dict = kwargs.pop('time')
+        if not isinstance(time_dict, dict):
+            raise ValueError('Wrong format. time must include '
                              'start and end keys for range. Must be a dict')
         else:
-            if 'start' in st_time_dict and 'end' in st_time_dict:
-                search_dict['create_time'] = {'$gte': st_time_dict['start'],
-                                                  '$lte': st_time_dict['end']}
+            if 'start' in time_dict and 'end' in time_dict:
+                search_dict['time'] = {'$gte': time_dict['start'],
+                                       '$lte': time_dict['end']}
             else:
-                raise ValueError('create_time must include start '
-                                     'and end keys for range search')
+                raise ValueError('time must include start '
+                                 'and end keys for range search')
     except KeyError:
         pass
 
     if search_dict:
-        header_objects = Header.objects(
-               __raw__=search_dict).order_by('-_id')[:limit]
+        br_objects = BeginRunEvent.objects(__raw__=search_dict).order_by('-_id')[:limit]
     else:
-        header_objects = list()
-    return header_objects
+        br_objects = list()
```

Comment:
can you fake this up better so that it always returns the same type of object?  Maybe just raise an exception if the search dict is empty?